### PR TITLE
5561 feat(admin) add a Proxy configuration to centreon test-lib 

### PR DESCRIPTION
**Pull request by MatthieuMan**
iAmLoggedInACentreonServerWithAConfiguredProxy() new method add, used as a background in TestConfigurationProxyContext

### CODE REVIEW

**Comment by MatthieuMan** [comment]
You meen beetwen $this->launchCentreonWebContainer('web_squid_simple') and $this->iAmLoggedIn() ?

**Comment by ganoze** [comment]
More likely juste after iAmLoggedIn(), to properly configure the proxy within the interface.

**Comment by MatthieuMan** [comment]
hum, I just pushed a PR on "Centreon feat(admin): Add a test configuration proxy button #5602", Do you prefer a refacto?

File: src/behat/CentreonContext.php [hunk 1]
```diff
@@ -349,6 +350,18 @@ public function launchCentreonWebContainer($name)
...
+    /**
+     * Set a proxy URL and port
+     */
+    public function setConfiguredProxy()
+    {
+        $proxyConfig = new ParametersCentreonUiPage($this);
+        $proxyConfig->setProperties(array(
+            'proxy_url'=> 'squid',
+            'proxy_port'=> '3128'
+            ));
```
Comment (ganoze) [inline src/behat/CentreonContext.php]: Does not seems PSR-2 compliant. Indentation should be made with the same level than $proxyConfig.

Comment (MatthieuMan) [reply src/behat/CentreonContext.php]: sorry

**Comment by Ridene** [comment]
Can you improve your commit message please ?

**Pull request merged**